### PR TITLE
Acount screen themeing tweaks

### PR DESCRIPTION
**Pull request by youngkidwarrior**

### CODE REVIEW

**Comment by youngkidwarrior** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/482?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#482** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/482?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#480** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/480?utm_source=stack-comment-icon" target="_blank">Graphite</a>: 1 other dependent PR ([#481](https://github.com/0xsend/sendapp/pull/481) <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/481?utm_source=stack-comment-icon" target="_blank">Graphite</a>)
* **#452** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/452?utm_source=stack-comment-icon" target="_blank">Graphite</a>: 1 other dependent PR ([#461](https://github.com/0xsend/sendapp/pull/461) <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/461?utm_source=stack-comment-icon" target="_blank">Graphite</a>)
* **#442** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/442?utm_source=stack-comment-icon" target="_blank">Graphite</a>: 1 other dependent PR ([#443](https://github.com/0xsend/sendapp/pull/443) <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/443?utm_source=stack-comment-icon" target="_blank">Graphite</a>)
* **#454** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/454?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `dev`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @youngkidwarrior and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/app/features/account/screen.tsx [hunk 1]
```diff
@@ -348,41 +362,28 @@ const NoTagsMessage = () => {
 
 const BottomButtonRow = () => {
   const tags = useConfirmedTags()
+  console.log('tags: ', tags)
```
Comment (0xBigBoss) [inline packages/app/features/account/screen.tsx:365]: can we drop this?

**Review by 0xBigBoss — APPROVED** [review]
just one thing

**Comment by youngkidwarrior** [comment]
### Merge activity

* **Jun 28, 9:31 PM EDT**: @youngkidwarrior started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/0xsend/sendapp/482).

**Pull request merged**